sdk/python/feast/constants.py

```diff
@@ -37,3 +37,6 @@
 
 # Default FTS port
 DEFAULT_FEATURE_TRANSFORMATION_SERVER_PORT = 6569
+
+# Environment variable for feature server docker image tag
+DOCKER_IMAGE_TAG_ENV_NAME: str = "FEAST_SERVER_DOCKER_IMAGE_TAG"
```

Comment:
Shouldn't `DOCKER_IMAGE_TAG_ENV_NAME` be something "feature server" specific?

Reply:
Why would we use different tags for different servers? Also, we could always rename this if the necessity arises. It won't break anything.